src/main/java/no/idporten/userservice/logging/TokenMasker.java

```diff
@@ -0,0 +1,24 @@
+package no.idporten.userservice.logging;
+
+public class TokenMasker {
+
```

Comment:
Kunne denne fra biblioteket blitt brukt istedenfor? https://github.com/felleslosninger/idporten-log-audit/tree/main/idporten-log-audit/src/main/java/no/idporten/logging/audit/masking

Reply:
teke i bruk jwttoken masker frå audit log lib i staden for.